contrib/test/integration/main.yml

```diff
@@ -34,6 +34,8 @@
     - integration
     - e2e
   tasks:
+    - name: Help prevent running out of memory
+      include: "swap.yml"
```

Comment:
Since this isn't a persistent swap device, if the ``setup`` tag is used to produce an AMI, it will not be available when used again for testing.  Avoid this by including the ``swap.yml`` twice, once under ``setup`` and again just before building.